pages/dashboard/index.vue

```diff
@@ -158,9 +158,9 @@ async function deleteSurvey(event: any, id: string) {
         </div>
 
         <DataTable v-else :value="surveys" table-style="width: 100%;">
-          <Column field="id" header="ID" style="width: 20%;" />
-          <Column field="title" header="Title" style="width: 50%" />
-          <Column field="workshop" header="Type" style="width: 20%">
+          <Column field="id" header="ID" :sortable="true" style="width: 20%;" />
```

Comment:
No need to sort by ID, since it is non deterministic

Reply:
I have removed the sorting for ID in the latest commit.